clang/lib/CodeGen/CGExprScalar.cpp

```diff
@@ -1035,7 +1050,7 @@ EmitIntegerTruncationCheckHelper(Value *Src, QualType SrcType, Value *Dst,
   }
 
   llvm::Value *Check = nullptr;
-  // 1. Extend the truncated value back to the same width as the Src.
+  // 1. Convert the Dst back to the same type as Src.
```

Comment:
This is not an improvement in the comment.

Reply:
You are correct 😃